packages/networking/src/networking.module.ts

```diff
@@ -24,13 +24,40 @@ export const NetworkingModule: ModuleInterface = {
         TelemetryModule,
     ],
     configurationDefinitions: [
+        /**
+         * Whether or not the request body converter interceptor is active.
+         */
         {
-            parameterName: NetworkingModuleKeyname + ".request_body_converter.is_active",
+            parameterName: NetworkingModuleKeyname + ".requestBodyConverter.isActive",
```

Comment:
we should create a task to unify the naming of parameters. I'm using "_" and you are using camelcase haha

Reply:
Yeah there is only one other place where there are "_" I'll remove it right now